.azure-pipelines/signing.yml

```diff
@@ -7,6 +7,42 @@ parameters:
 
 steps:
 - ${{ if parameters.isWindows }}:
+  - task: PowerShell@2
+    inputs:
+      targetType: 'inline'
+      script: |
+        $ErrorActionPreference = 'Stop'
```

Comment:
Could you please split up the logic to several cmdlets (probably - retrieval of signtool path and signatures removal), and add error handling for signtool path retrieval also?
Would it also make sense to move this to the ps1 file instead then?

Reply:
Yeap, I've updated the logic of the script according to your suggestion